python/tensorflowjs/converters/tf_saved_model_conversion_v2.py

```diff
@@ -391,3 +405,43 @@ def convert_tf_hub_module(module_path, output_dir,
     # Clean up the temp files.
     if os.path.exists(frozen_file):
       os.remove(frozen_file)
+
+
+def convert_tf_hub_module(module_handle, output_dir,
+                          signature='default', saved_model_tags='',
```

Comment:
should this have a default value?

Reply:
There is actually no reasonable default, but will set it to 'serve' for now.